ambry-api/src/main/java/com/github/ambry/quota/QuotaChargeCallback.java

```diff
@@ -51,22 +50,23 @@ public void charge(long chunkSize) throws RouterException {
           if (throttlingRecommendation != null && throttlingRecommendation.shouldThrottle() && shouldThrottle) {
             if (quotaManager.getQuotaMode() == QuotaMode.THROTTLING
                 && quotaManager.getQuotaConfig().throttleInProgressRequests) {
```

Comment:
so if the throttleInProgressRequests is false, we are not going to throttle any requests, including ttl update and delete?